README.md

```diff
@@ -49,70 +68,135 @@ const client = new Realtime(ABLY_API_KEY);
 const spaces = new Spaces(client);
 ```
 
-Create a new space instance and have members enter that space:
-
-```ts
-const space = spaces.get('mySpace');
-space.enter({
-  username: 'Example User',
-  avatar: 'https://example.com/users/1234.png',
-});
-```
+### Space membership
 
-Subscribe to updates from other members in the space. The `membersUpdate` event will fire when a member's connection status updates, e.g. when a member joins or leaves the space:
+A space is the virtual area of an application you want to monitor, such as a web page, or slideshow. Create a space and subscribe to events for when clients enter and leave it. Space membership is used to build avatar stacks and display which members are online within a space.
 
 ```ts
+// Create a new space
+const space = spaces.get('demoSlideshow');
+
+// Register a listener to subscribe to events of when users enter or leave the space
 space.on('membersUpdate', (members) => {
   console.log(members);
 });
-```
 
-### Example payload returned
+// Enter a space, publishing a memberUpdate event, including optional user data
+space.enter({
+  username: "Claire Lemons",
+  avatar: "https://slides-internal.com/users/clemons.png",
+});
+```
 
-1. When a member has entered the space via `space.enter()`:
+The following is an example `membersUpdate` event received by subscribers when a user enters a space:
 
 ```json
 [
   {
-    "clientId": "user1234",
+    "clientId": "clemons@slides.com",
     "isConnected": true,
     "lastEvent": {
       "name": "enter",
       "timestamp": 1677595689759
     },
     "location": null,
     "profileData": {
-      "username": "Example User",
-      "avatar": "https://example.com/users/1234.png"
+      "username": "Claire Lemons",
+      "avatar": "https://slides-internal.com/users/clemons.png"
     }
   }
 ]
 ```
 
-2. When a member has left the space in one of the following ways:
+### Location updates
 
-- called `space.leave()`
-- has closed the tab
-- abruptly disconnected from the internet for more than 2min (which is configurable via [`offlineTimeout`](https://github.com/ably-labs/spaces/blob/main/docs/class-definitions.md#offlinetimeout):
+Locations enable you to track where clients are within an application. Subscribe to all location updates, those for a specific location, such as a single UI element, or a specific client.
+
+```ts
+// Register a listener to subscribe to events of when users change location
+space.locations.on('locationUpdate', (update) => {
+  console.log(update);
+});
+
+// Publish locationUpdate event with a client's location when they select a UI element or change web pages
+space.locations.set({slide: '3', component: 'slide-title'});
+
+// Create a tracker to only publish locationUpdate events for a specific user using their clientId
```

Comment:
More of a suggestion: it may be worth explicitly adding 'option 1: ...', 'option 2:..' etc to each way of subscribing to locations so readers don't mistake the snippet as the thing they'd paste directly to work with.

Reply:
Hmm... I get your point, but people may want to use a combination of them all?

Reply:
True, but that's still accomplished with options, no?